cumulusci/tasks/push/tasks.py

```diff
@@ -237,12 +237,12 @@ def _init_task(self):
     def _init_options(self, kwargs):
         super(SchedulePushOrgList, self)._init_options(kwargs)
 
-        neither_file_option = "orgs" not in self.options and "csv" not in self.options
-        both_file_options = "orgs" in self.options and "csv" in self.options
-        if neither_file_option or both_file_options:
-            raise TaskOptionsError(
-                "Please call this task with either the `orgs` or `csv` option."
-            )
+        # neither_file_option = "orgs" not in self.options and "csv" not in self.options
+        # both_file_options = "orgs" in self.options and "csv" in self.options
+        # if neither_file_option or both_file_options:
+        #     raise TaskOptionsError(
+        #         "Please call this task with either the `orgs` or `csv` option."
+        #     )
```

Comment:
I see this is related to what you were asking about in standup. I think you should go ahead and fix _init_options for SchedulePushOrgQuery as part of this PR.

Reply:
So initially I was going to remove the additional option restriction from `SchedulePushOrgList`, but added it to `SchedulePushOrgQuery` instead let me know if you agree with this approach.